SAS/mi_mar_regression.qmd

```diff
@@ -0,0 +1,100 @@
+---
+
+title: "Multiple Imputaton: Linear Regression in SAS"
+
+---
+
+## Multiple Imputation
+
+#### Input dataset preparation before multiple imputation
```

Comment:
You might want to make your subsequent headings a little bigger, for e.g. using two hashes (##) instead of four?